performance_test/src/data_running/data_runner.hpp

```diff
@@ -151,7 +152,10 @@ class DataRunner : public DataRunnerBase
         m_time_reserve_statistics.add_sample(reserve.count());
         m_lock.unlock();
       }
-      if (m_ec.rate() > 0 && m_run_type == RunType::PUBLISHER) {
+      if (m_ec.rate() > 0 &&
+          m_run_type == RunType::PUBLISHER &&
+          // Relays should never sleep.
+          m_ec.roundtrip_mode() != ExperimentConfiguration::RoundTripMode::RELAY) {
```

Comment:
Since, this sleep will add latency in the relay mode correct?

Reply:
Yep.